ocrd_kraken/binarize.py

```diff
@@ -17,11 +17,6 @@ class KrakenBinarize(Processor):
     def executable(self):
         return 'ocrd-kraken-binarize'
 
-    def setup(self):
-        self.logger = getLogger('processor.KrakenBinarize')
-        assert_file_grp_cardinality(self.input_file_grp, 1)
-        assert_file_grp_cardinality(self.output_file_grp, 1)
-
```

Comment:
I understand the changes regarding the file group cardinality checks but why have you removed the setup method completely?

Reply:
Oops! I should have kept the logger. Thanks for spotting!